client/src/legacy/SudoModePasswordField/SudoModePasswordFieldEntwine.js

```diff
@@ -47,8 +49,14 @@ jQuery.entwine('ss', ($) => {
      * This is here instead of inside the react component because in a pure react form the success handler
      * should result in re-rendering the react form with editable fields, instead of reloading the whole page.
      */
-    reloadPage() {
-      this.closest('.cms-container').reloadCurrentPanel();
+    reloadSection() {
+      const gridField = this.closest('.ss-gridfield');
+      if (gridField.length) {
```

Comment:
I'm not convinced this is a robust way to check if the field is for a gridfield or not. Probably better to add a `data-gridfield="true"` or similar attribute when it's for a gridfield instead.
You can set attributes on the field using `$field->setAttribute()`